Client/Frontend/Fakespot/Components/CardContainer.swift

```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Shared
+import UIKit
+
+class CardContainer: UIView, ThemeApplicable {
+    struct UX {
+        static let verticalPadding: CGFloat = 8
+        static let horizontalPadding: CGFloat = 8
+        static let cornerRadius: CGFloat = 8
+        static let shadowRadius: CGFloat = 14
+        static let shadowOpacity: Float = 1
+        static let shadowOffset = CGSize(width: 0, height: 2)
+    }
+
+    // MARK: - Properties
+
+    // UI
+    private lazy var rootView: UIView = .build { _ in }
+    lazy var containerView: UIView = .build { _ in }
```

Comment:
nit, possible?
```suggestion
    private lazy var containerView: UIView = .build { _ in }
```

Reply:
Unfortunately not because otherwise we cannot add content to the card (and constraint the content to it's container).